qcelemental/models/common_models.py

```diff
@@ -33,6 +33,15 @@ class DriverEnum(str, Enum):
     hessian = 'hessian'
     properties = 'properties'
 
+    def derint(self):
```

Comment:
Can we make this name more verbose, and `properties` should return `None`.

Reply:
`to_integer` or `derivative_int`?

for `properties` would be nice to have a response that can do math that doesn't require error handling.

Reply:
Either one on the names. Ok on `properties=0`.